packages/firestore-compat/src/index.console.ts

```diff
@@ -98,6 +102,15 @@ export class Firestore extends FirestoreCompat {
       new MemoryPersistenceProvider()
     );
   }
+
+  INTERNAL = {
+    delete: () => this.terminate(),
+    count: (query: ExpQuery<unknown>) => {
+      return getCountFromServer(query).then((response: AggregateQuerySnapshot<AggregateSpec>) => {
```

Comment:
I think the Promise returned from getCountFromServer() needs a .catch() too. See what happens if the promise returned from getCountFromServer() is rejected. I don't think the promises are chained properly.

Reply:
you shouldn't need it cause the consumer should have the catch to handle the error state in the promise chain, but just in case, added the catch to rethrow the error for the consumer to handle.

Reply:
Ahh I think you're right. Please remove the unnecessary catch(). Sorry for the false alarm... TypeScript is not my strongest language.

Reply:
removed it 😄